test/server/runner-test.js

```diff
@@ -859,6 +859,32 @@ describe('Runner', () => {
         });
     });
 
+    describe('.requestHooks', () => {
+        it('Should raise an error for multiple ".requestHooks" method call', () => {
```

Comment:
Add a functional test to the `test/functional/fixtures/api/es-next/request-hooks/test.js` file. The test should check the hook execution order (the first - common request hooks, the next - test's request hooks).